tracex_project/extraction/templates/result.html

```diff
@@ -98,5 +116,7 @@ <h1>Success! The TracEX pipeline produced the following results:</h1>
     <a href="{% url 'landing_page' %}">
         <button class="menu_button">Return to TracEX Menu</button>
 
+    <script src="{% static '/extraction/js/pop_up_xes.js' %}" defer></script>
```

Comment:
script should be loaded in the header of the html, for reference look at the filter_journey html